DEVELOPMENT.md

```diff
@@ -74,7 +74,7 @@ requires that the `federation-demo` project is running:
 Once the subgraphs are up and running, run Apollo Router with this command:
 
 ```shell
-cargo run -- -s ./examples/local.graphql
+cargo run --release -- -s ./examples/local.graphql -c examples/config-jaeger.yml
```

Comment:
why --release in development?

Reply:
Because everybody (including me) is ending up asking "heyyy why is you query planning running in 50ms while mine took 200ms" or something :grin: I kinda expect to know they can remove `--release` if they feel it's not needed.

`--release` is explicit which is nice but debug is implicit and that's why we always feel surprised I think.

Note also that this command is not for running tests but to run the server and check if it work.

Reply:
Let's add --debug so people notice it's intentional then, I don't think devs shall work against --release, they lose failfast_debug!() and the compile time isn't great ^^

Reply:
failfast_debug only works if you explicitly add the features. It's not included in debug by default :sweat_smile:

Reply:
```
[0] [12:13:13] ~/r/router cecton-start-with-jaeger > c build --debug
error: Found argument '--debug' which wasn't expected, or isn't valid in this context

USAGE:
    cargo build [OPTIONS]

For more information try --help
```

Reply:
There is no --debug. You're trying to confuse me

Reply:
Oh I'm sorry I would have expected it to exist :(

Yeah I'm not in favor of --release, but I don't mind that much!

Reply:
How dare you agree with me xD

Reply:
See, it happens 😄